string data_path
string analy_type